### PR TITLE
Feature/40 task running doc

### PR DESCRIPTION
**Pull request by kevinmpowell**
Addresses: https://github.com/EightShapes/esds-project/issues/40

This diff is dense since I moved so much stuff around. I recommend you review:

* https://github.com/EightShapes/esds-library/blob/feature/40-task-running-doc/README.md
* https://github.com/EightShapes/esds-library/blob/feature/40-task-running-doc/components/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+# EightShapes Design System
 
 ## Repository Structure
-The EightShapes Design system is a monorepo containing design system components, tokens, icons and visual assets. Each of these assets is published as an independently versioned node package. 
+The EightShapes Design system is a monorepo containing design system components, tokens, icons and visual assets. Each of these assets is published as an independently versioned node package.
 
-The dependencies between these assets are managed using [lerna](https://lerna.js.org). 
+The dependencies between these assets are managed using [lerna](https://lerna.js.org).
+
+**IMPORTANT** Follow the [lerna installation instructions](./documentation/lerna.md) before running any commands in this repository.
```
Comment (nathanacurtis) [inline README.md]: I'm not a fan of **IMPORTANT**s littered everywhere. A rewrite could be:

> Before running commands in this repository, [install lerna](./documentation/lerna.md).

File: README.md [hunk 2]
```diff
@@ -12,57 +14,19 @@ To view a dependency tree of the repository, in the _root_ of the repo run:
...
 
-##### [stylelint](https://stylelint.io)
-Enforces `.scss` coding styleguide
+### esds-icons
+EightShapes Design System Icons are svg assets used throughout design system components. [Icons README](./esds-icons/README.md)
 
-##### [stylelint-config-prettier](https://github.com/prettier/stylelint-config-prettier)
-Turns off all rules that are unnecessary or might conflict with prettier.
+### components/[component]
+Components are versioned independently and distributed as individual node packages. [Components README](./components/README.md)
```
Comment (nathanacurtis) [inline README.md]: This content could be more concise, such as:

> This repository includes many packages across folders:
> * [EightShapes Design System Tokens](./esds-tokens/README.md), key/value pairs representing visual style properties.

File: components/README.md [hunk 3]
```diff
@@ -1,80 +1,70 @@
-# Creating a new component package
+# EightShapes Design System Components
 
-## 1. Set up the component build pipeline
-Be sure you're in the `/components` directory of the repo: `cd components`, then run:
+
+## Running component dev environments
+**IMPORTANT** Follow the [lerna installation instructions](../documentation/lerna.md) before running any commands in this repository.
```
Comment (nathanacurtis) [inline components/README.md]: Consider using the same suggestion here as with the other README.

File: components/README.md [hunk 4]
```diff
@@ -1,80 +1,70 @@
...
+For performance reasons, the default dev environment does not build assets for IE11. To start an IE 11 compatible dev environment run:
+
+```
+cd components/[component-name]
+npm start:legacy
 ```
 
-Becomes:
+---
+## Dev Environment Dependency Rationale
```
Comment (nathanacurtis) [inline components/README.md]: Why not just title this section **Dev Dependencies**?

File: components/README.md [hunk 5]
```diff
@@ -1,80 +1,70 @@
...
+### [@babel/core](https://github.com/babel/babel/tree/master/packages/babel-core)
```
Comment (nathanacurtis) [inline components/README.md]: So many headers makes this content hard to read. Would a simpler list format like this work better?

> * [@babel/core](https://github.com/babel/babel/tree/master/packages/babel-core) compiles ES6 source code to ES5 for IE11 Compatibility

Comment (kevinmpowell) [reply components/README.md]: I had that exact format initially and found _it_ hard to read. My cap score is low on this format. Your call.

Comment (nathanacurtis) [reply components/README.md]: Maybe bold, semicolon, comma or line break could separate package and description to improve contrast? My instinct suggests a simpler structure than headers is desirable for a long list.

Comment (kevinmpowell) [reply components/README.md]: I just compared my local markdown editor (where I created these) and Github's formatting, and I agree. The headers don't work on Github:

My local markdown editor:
Screen Shot 2020-02-27 at 3 01 39 PM

Github:
Screen Shot 2020-02-27 at 3 01 44 PM

I need to get a local editor that imports Github's CSS...I'll reformat.

Comment (kevinmpowell) [reply components/README.md]: Where I landed:
image

**Pull request merged**